### ISSUE TITLE
ion-img-cache-bg: not working with error messages if I use linear-gradient 

### ISSUE THREAD

**Issue by almothafar** [vitaliy-bobrov/ionic-img-cache#9]
Hello 

I'm using the `linear-gradient()` with `background-image` the problem is I get a lot of error messages in the console, and seems cache not working as well.

.image-bg {
    background: linear-gradient(rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.60)), url('img-url-here')  center center no-repeat;
}

not sure if there is another properties for this, but currently I have with `linear-gradient`, I get the following prints in the console: 

```
INFO: Background image URL: rgba(0, 0, 0, 0.6), rgba(0, 0, 0, 0.6)), url(/api/image/2
imgcache.js:468 GET http://localhost:8100/rgba(0,%200,%200,%200.6),%20rgba(0,%200,%200,%200.6)),%20url(/api/image/2 404 (Not Found)Private.FileTransferWrapper.download @ imgcache.js:468ImgCache.cacheFile @ imgcache.js:640ImgCache.cacheBackground @ imgcache.js:838(anonymous function) @ ionic-img-cache.js:113(anonymous function) @ imgcache.js:820(anonymous function) @ imgcache.js:710
imgcache.js:46 ERROR: Image rgba(0, 0, 0, 0.6), rgba(0, 0, 0, 0.6)), url(/api/image/2 could not be downloaded - status: 404
imgcache.js:46 ERROR: Download error source: rgba(0, 0, 0, 0.6), rgba(0, 0, 0, 0.6)), url(/api/image/2
imgcache.js:46 ERROR: Download error target: /ionic-img-cache/9d04e5c7c904a9f0de5ecd3e4809340bd5c42a60.2
imgcache.js:46 ERROR: Download error code: 0
```

Edit: sorry seems you can't do anything with this?

related to : https://github.com/chrisben/imgcache.js

**Comment by vitaliy-bobrov** [comment]
@almothafar, thank you for reporting, I'll try to look on that

**Comment by vitaliy-bobrov** [comment]
@almothafar, Closing this one, because probplem is inside imgcache.js library.

**Comment by almothafar** [comment]
thanks @vitaliy-bobrov, I saw your PR, simple solution, but seems the owner is not active, last 1 year he almost empty in activity, I suggest to adopt imgcache.js , you maybe need to fork it and use forked one, he even planned to version 2 and abandoned it.

See another pull requests also, there could be issue with API 23 seems, anyway just suggestion.

**Comment by vitaliy-bobrov** [comment]
Let wait for the imgcache maintainer answer

**Comment by almothafar** [comment]
@vitaliy-bobrov  any news from imgcache maintainer?

**Comment by almothafar** [comment]
Hello @vitaliy-bobrov  finally there is a new release to fix this issue from imgcache.js , version 1.1.1.

**Comment by vitaliy-bobrov** [comment]
@almothafar, Published version 1.1.2 with updated imgcache version as dependency
